src/wombats/handlers/helpers.clj

```diff
@@ -24,3 +25,102 @@
                                             :code code}
                                      (not (nil? field-error))
                                      (merge {:field-error field-error}))))))
+
+(defn- format-link
+  [link]
+  (let [query (:query (url link))
+        page-number-string (get query "page")]
+    {:query query
+     :page-number (when page-number-string
+                    (Integer/parseInt page-number-string))
+     :link link}))
+
+(defn- format-links
+  [links]
+  (apply merge
+   (map (fn [link]
+          (let [[ln rel] (clojure.string/split link #";")]
+            (when (and ln rel)
+              {(keyword (last (re-find #"rel=\"(.*)\"" rel)))
+               (format-link (last (re-find #"<(.*)>" link)))})))
+        links)))
+
+(defn parse-link-headers
+  "Parses the link header into a usable data structure.
+
+  Ex: [{:query {}
+        :page-number 0
+        :link \"http://...\"}]
+
+  NOTE: This could be used put into a cljc file and used on the client as well"
+  [context]
+  (let [link-header (get-in context [:request :headers "link"])]
+    (when link-header
+      (-> link-header
+          (clojure.string/split #",")
+          (format-links)))))
+
+(defn- format-url
+  [base uri]
+  (fn [query]
+    (str base uri "?" (clojure.string/join
+                       "&"
+                       (map (fn [[k v]]
+                              (str (name k) "=" v))
```

Comment:
You'll control the input on this? Just asking since it doesn't escape anything....